File: DisplaBackend/DisplaBackend/Models/ComprobanteCliente.cs
using System;
using System.Collections.Generic;

namespace DisplaBackend.Models
{
    public partial class ComprobanteCliente
    {
        public ComprobanteCliente()
        {
            ComprobanteItem = new HashSet<ComprobanteItem>();
            VentaVirtual = new HashSet<VentaVirtual>();
            VentaVirtualMovimientos = new HashSet<VentaVirtualMovimientos>();
        }

        public int Id { get; set; }
        public int IdCliente { get; set; }
        public int IdTipoComprobante { get; set; }
        public DateTime Fecha { get; set; }
        public int Numero { get; set; }
        public int Sucursal { get; set; }
        public string Letra { get; set; }
        public decimal? TasaIva { get; set; }
        public decimal? MontoIibb { get; set; }
        public decimal? MontoTseh { get; set; }
        public decimal MontoTotal { get; set; }
        public decimal SubTotalFactura { get; set; }
        public decimal? PorcentajeDtoGral { get; set; }
        public decimal? MontoIvari { get; set; }
        public DateTime? FechaAnulado { get; set; }
        public int? IdUsuario { get; set; }
        public string Observaciones { get; set; }

        public virtual Cliente IdClienteNavigation { get; set; }
        public virtual TipoComprobante IdTipoComprobanteNavigation { get; set; }
        public virtual AspNetUsers IdUsuarioNavigation { get; set; }
        public virtual ICollection<ComprobanteItem> ComprobanteItem { get; set; }
        public virtual ICollection<VentaVirtual> VentaVirtual { get; set; }
        public virtual ICollection<VentaVirtualMovimientos> VentaVirtualMovimientos { get; set; }
    }
}

File: DisplaFrontend/src/app/model/comprobanteCliente.ts
import { Cliente } from './Cliente';
import { ComprobanteItem } from './comprobanteItem';
import { TipoComprobante } from './tipoComprobante';
import { Usuario } from './usuario';
import { VentaVirtual } from './ventaVirtual';

export interface ComprobanteCliente {
    Id: number,
    IdCliente: number,
    IdClienteNavigation: Cliente,
    IdTipoComprobante: number,
    IdTipoComprobanteNavigation: TipoComprobante,
    Fecha: Date,
    Numero: number,
    Sucursal: number, 
    Letra: string,
    TasaIva: number, 
    MontoIIBB: number,
    MontoTSEH: number,
    MontoTotal: number,
    SubtotalFactura: number,
    PorcentajeDtoGral: number,
    Observaciones: string,
    MontoIVARI: number,
    IdUsuario: number,
    IdUsuarioNavigation: Usuario,

    ComprobanteItem: ComprobanteItem[],
    VentaVirtual: VentaVirtual[]
}